website/docs/terms/hover-terms.md

```diff
@@ -0,0 +1,136 @@
+---
+cte: 
+  displayText: CTE
+  hoverSnippet: A Common Table Expression (CTE) is a temporary result set that can be used in a SQL query. You can use CTEs to break up complex queries into simpler blocks of code that can connect and build on each other.
+
+dag:
+  displayText: DAG
+  hoverSnippet: A DAG is a Directed Acyclic Graph, a type of graph whose nodes are directionally related to each other and don’t form a directional closed loop.
+
+data-extraction:
+  displayText: data extraction
+  hoverSnippet: Data extraction is the process by which data is retrieved from multiple sources, often varying in volume and structure.
+
+data-lake:
+  displayText: data lake
+  hoverSnippet: A data lake is a data management system used for storing large amounts of data in in its raw, native form as files. Data lakes can store any type of data—structured, semi-structured, unstructured—in one centralized place.
+
+data-lineage:
+  displayText: data lineage
+  hoverSnippet: Data lineage provides a holistic view of how data moves through an organization, where it’s transformed and consumed.
+
+data-warehouse:
+  displayText: data warehouse
```

Comment:
Does this refer to the links when clicking the terms themselves on a page? If so - yes, these hover snippets will no longer be linkable, only the hover snippet functionality remains in place.
![image](https://github.com/user-attachments/assets/b1293629-8647-43da-b981-38cdf919eb7f)

If term URLs are hit directly, they will either redirect to the migrated WWW blog page if that term was migrated, or redirected to the WWW blog index if that term wasn't migrated.

I did update two files where we were linking to term pages as Docusaurus was flagging them as broken links. But even if any links are still out there, they will redirect rather than 404ing when hit